integration-tests/src/main/java/org/apache/polaris/service/it/env/PolarisApiEndpoints.java

```diff
@@ -49,4 +49,8 @@ public URI managementApiEndpoint() {
   public String realm() {
     return realm;
   }
+
+  public String realmHeader() {
```

Comment:
Maybe rename to `realmHeaderName()` + `realmId()` (latter for consistency w/ `Server`)?